lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -829,17 +899,29 @@ def remove_toolitem(self, name):
                     self._groups[group].remove(toolitem)
         del self._toolitems[name]
 
+    def set_flow(self, flow):
```

Comment:
Actually, looking back at my code I did actually implement this :smile:, you can change the flow at run time, it works!  You can change it any moment :smile:.  Should I keep it?  If so I think I should turn it into a property via getter/setter.

Also about putting this in ``ToolContainerBase``, that feels wrong (perhaps why I didn't do it in the first place).  As far as I can see, the ``flow`` attribute should only apply to toolbars and not other types of ``ToolContainer``s (which we will no doubt design in the future, like menus).  Therefore I would suggest creating the intermediary class ``ToolbarBase``.  As we now use ``super`` with these classes it makes it very easy (and minor) to implement this change.  What do you think?